Source/GKRect.m

```diff
@@ -4,7 +4,7 @@
 #import "GKRect.h"
 
 @implementation GKRect
-@dynamic midX, midY;
+@dynamic midX, midY, maxX, maxY, origin, size;
```

Comment:
No they’re need here because of the GKRect interface. Xcode threw errors so that’s why I put them there